css-view-transitions-1/Overview.bs

```diff
@@ -162,7 +164,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			Note: The value <css>none</css> is invalid as a <<custom-ident>>.
 	</dl>
 
-	The [=document element=] participates in a view transition by default using the following style in the [=user-agent origin=]:
+	The [=document element=] participates in a view transition by default using the following rule,
+	added to the [=HTML user agent style sheet=]:
```

Comment:
If the style never changes, and does no harm always-being-there, I've added it to the "HTML user agent style sheet"